### DISCUSSION TITLE
how to see the `:Inspect`'s result text, in :messgaes command list

### DISCUSSION THREAD

**Discussion by xzbdmw** [folke/noice.nvim#719] [Q&A] [answered]
Currently it won't be included in message history, but I want to yank highlight group from then.

**Comment by ilias777** [comment, marked as answer]
After the command `:Inspect` you can use the command `:NoiceAll`.

It opens a split view and you can yank from there.

https://github.com/folke/noice.nvim/assets/39212564/325617c2-b190-4432-883b-00d7ced9fe54
